File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/src_delta_all_type_no_partition.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_delta_all_type_no_partition {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("delta")
      .load("dbfs:/Prophecy/qa_data/delta/all_type_no_partition")

}

File: testprojects/python_main/pipelines/PythonGenericUse/code/pythongenericuse/graph/src_delta_all_type_no_partition.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from pythongenericuse.config.ConfigStore import *
from pythongenericuse.udfs.UDFs import *

def src_delta_all_type_no_partition(spark: SparkSession) -> DataFrame:
    return spark.read.format("delta").load("dbfs:/Prophecy/qa_data/delta/all_type_no_partition")
